infra/bridge/src/controllers/google.js

```diff
@@ -98,7 +98,10 @@ router.post('/verify', googleVerify, async (req, res) => {
     site: {
       siteName: 'google.com',
       userId: {
-        verified: true
+        raw: String(userDataResponse.body.id)
+      },
+      username: {
+        raw: userDataResponse.body.email
```

Comment:
@nick we probably don't want to store this in IPFS, do we?

Reply:
yea, no email in IPFS